Framework/TestWorkflows/src/o2DiamondWorkflow.cxx

```diff
@@ -8,15 +8,28 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 #include "Framework/ConfigParamSpec.h"
+#include "Framework/CompletionPolicy.h"
+#include "Framework/DeviceSpec.h"
 #include <vector>
 using namespace o2::framework;
 void customize(std::vector<ConfigParamSpec> &options) {
-  options.push_back(o2::framework::ConfigParamSpec{"anInt", VariantType::Int, 1, {"an int option"}});
-  options.push_back(o2::framework::ConfigParamSpec{"aFloat", VariantType::Float, 2.0f, {"a float option"}});
-  options.push_back(o2::framework::ConfigParamSpec{"aDouble", VariantType::Double, 3., {"a double option"}});
-  options.push_back(o2::framework::ConfigParamSpec{"aString", VariantType::String, "foo", {"a string option"}});
-  options.push_back(o2::framework::ConfigParamSpec{"aBool", VariantType::Bool, true, {"a boolean option"}});
-};
+  options.push_back(ConfigParamSpec{"anInt", VariantType::Int, 1, {"an int option"}});
+  options.push_back(ConfigParamSpec{"aFloat", VariantType::Float, 2.0f, {"a float option"}});
+  options.push_back(ConfigParamSpec{"aDouble", VariantType::Double, 3., {"a double option"}});
+  options.push_back(ConfigParamSpec{"aString", VariantType::String, "foo", {"a string option"}});
+  options.push_back(ConfigParamSpec{"aBool", VariantType::Bool, true, {"a boolean option"}});
+}
+
+void customize(std::vector<CompletionPolicy> &policies) {
+  auto matcher = [](DeviceSpec const &device) -> bool {
+    return device.name == "D";
+  };
+  auto policy = [](gsl::span<PartRef const> const &inputs) -> CompletionPolicy::CompletionOp {
+    return CompletionPolicy::CompletionOp::Process;
+  };
```

Comment:
since this is a "tutorial test" maybe it would be good adding a comment of what is achieved.